packages/designmanual/stories/organisms/NotionBlockExample.tsx

```diff
@@ -69,6 +74,20 @@ class NotionBlockExample extends Component {
             description: 'Velger embeded type innhold for forklaringsboksen',
           },
 
+          {
+            name: 'hideIconsAndAuthors ',
+            type: 'boolean',
+            default: '',
+            description: 'Fjerner lisensikonene og forfatterne',
+          },
+
+          {
+            name: 'inSearchResults',
```

Comment:
Denne finnes kun i designmanualen. Er det meininga at den skal kunne brukes også i ndla-frontend eller må dette kodes tilsvarende der? Hadde vore mest praktisk om det var en prop på notion-komponenten.
Er ikkje veldig glad for navn som forteller kvar en ting er brukt, men heller beskriver kva som skjer. hideIconsAndAuthors er bra for det er beskrivende. Denne burde kanskje heller hett adjustSizeToFitWiderPage eller nokke sånt.